### DISCUSSION TITLE
3.7.x Subscription for sync tax webooks

### DISCUSSION THREAD

**Discussion by TomSteer** [saleor/saleor#10760] [Q&A] [answered]
We are currently testing the calculate taxes webhook but now need to use the subscription for it. However, the subscription event doesn't look like it has been included in a release yet ([See this merge](https://github.com/saleor/saleor/pull/10433)), although it's mentioned in the docs. Is there a plan to release this in a 3.7.x release soon? 

Thanks

**Comment by designagencytom** [comment]
+1 for this

**Comment by CossackDex** [comment, marked as answer]
Hi, @TomSteer and @designagencytom,

Mentioned PR will be merged and released in the next minor release, 3.8.

**Reply by designagencytom** [reply]
Thanks @CossackDex - Do you have any idea of a rough ETA on this please, just so we can plan around this?
